examples/potree_25d_map.html

```diff
@@ -49,7 +49,7 @@
                 viewerDiv = document.getElementById('viewerDiv');
                 viewerDiv.style.display = 'block';
 
-                debugGui = new dat.GUI({ width: 400 });
+                debugGui = new dat.GUI();
```

Comment:
In the GUI, movement speed is supposed to have what effect? I can't see reactions

Reply:
Sorry, it hadn't been rebased!